site/src/entries/chemical-extract.md

```diff
@@ -8,12 +8,23 @@ TODO intro paragraph and how to begin to add
 
 ## Required Information
 
-- TODO
+- **Nickname:** When you have several Isolates, it may be useful to give them short names to help distinguish them. Please follow your instructor's guidelines on how to name entries meaningfully.
+- **Date:** Date: The date the isolate was first created.
+
 
 ## Optional Information
 
-- TODO
+- **Isolate:** Click the "add Isolate" button to link an isolate to this entry. This field is automatically populated for you if you click "Add Eukaryotic Test" to an existing isolate.
+- **Media Used and Concentration of Media Used:** The base media used to cultivate the plate and its concentration. For example, commonly when the Media Used is TSA, the Concentration is 10%. Confirm with your instructor the concentration of your media.
+- **Additives and Concentration of Additives:** Additives to the media, such as antifungals, and their concentration (in ug/mL). Confirm with your instructor the additives to your media and their concentration.
+- **Media Modification:** A description of other modifications you made to your media, such as adding honey, essential oils, cinnamon, or so on. Many students like to test the effect of such treatments on their isolates- that treatment would be recorded here.
+- **Incubation Temperature and Duration:** The temperature (in Celsius) and the duration (in hours) that the plate was incubated.
+- **Solid or Liquid Culture:** Select solid or liquid culture. Note that when you change your selection, the following prompts will change with it. When the selection is solid, you will be asked for the number of plates used. When the selection is liquid, you will be asked whether or not resin was used.
+- **Drying Agent Used:** TODO
+- **Mass of Crude Extract:** TODO
+- **Experimental Design:** Give details describing your experimental procedure.
```

Comment:
Add language (like we did in Media used and Nickname) to check with instructor about their preferences

Reply:
is this regarding experimental design or all 22-25?